app/Http/Controllers/EmailController.php

```diff
@@ -0,0 +1,99 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\GenericModel;
+use Illuminate\Http\Request;
+
+use App\Http\Requests;
+use App\Helpers\MailSend;
+
+class EmailController extends Controller
+{
+
+    /**
+     * Send emails to list of user Ids
+     * @param Request $request
+     * @return \Illuminate\Http\JsonResponse
+     */
+    public function sendEmail(Request $request)
+    {
+        $to = $request->input('to');
+        $message = $request->input('message');
+        $subject = $request->input('subject');
+
+        $errors = [];
+
+        $recipients = $this->validateInput($to, $message, $errors);
```

Comment:
`validateInput` is a bad name here since it returns profile data - should be `loadProfiles`